### ISSUE TITLE
Best result

### ISSUE THREAD

**Issue by azizessid** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#236]
HI everyone,

I have 13 images of 1 class, 11 in the images/train et 2 in images/test, they all are picture of the same objects in different condition, background luminosity but the object is always in the same position.

After creating the model with 500 steps (the loss was constantly under 0.2), I did try it with the object_detection_webcam script and it does work but with a low match in most case ~70% and [20% - 99%].

But the object_detection_image with imagea I used in the training session I only have 8%-15%.

I want to know why and how to improve that score.

PS : The object is a pen

Thank you for reading :)
